Tests/TestingApps/PaywallsTester/PaywallsTester/Extensions/ArrayExtensions.swift

```diff
@@ -16,3 +17,42 @@ public extension Sequence {
     }
 
 }
+
+#if PAYWALL_COMPONENTS
+public extension Array where Element == PaywallComponent {
```

Comment:
NItpick but should this be internal? (doesn't matter much since it's for the test app)

Reply:
Done, and moved these to their own file (because dictionary isn't an array, and because these extensions are so specific, not generic array extensions).